client/lib/abtest/active-tests.js

```diff
@@ -8,6 +15,21 @@ module.exports = {
 		defaultVariation: 'disabled',
 		allowExistingUsers: true,
 	},
+	// `browserNotificationsPreferences` controls whether or not users see the
+	// preference to enable browser notifications in /me/notifications;
+	// any users with `browserNotifications` enabled will also see the preference;
+	// this is a temporary test just to allow us to ramp up the load gradually so
+	// that make sure things will be smooth when we go to launching browser
+	// notifications for 100% of users
+	browserNotificationsPreferences: {
+		datestamp: '20160801',
+		variations: {
+			disabled: 0,
```

Comment:
I am not sure how `0` would be handled here - It'd be more clear if we wrote 60: then it'd be expressed as percentage (60%-40%). Even though allocations do not have to add up to 100, it's just easier to read that way.